pkg/components/strawberry_controller.go

```diff
@@ -64,6 +65,9 @@ func NewStrawberryController(
 		if resource.Spec.StrawberryController.Image != nil {
 			image = *resource.Spec.StrawberryController.Image
 		}
+		if len(resource.Spec.StrawberryController.Tolerations) != 0 {
+			tolerations = resource.Spec.StrawberryController.Tolerations
+		}
```

Comment:
No point of this check, just pass `resource.Spec.StrawberryController.Tolerations` in newMicroservice